Dockerfile

```
@@ -20,6 +20,10 @@ COPY . .
 
 RUN pip install -e . --no-cache-dir
 
+RUN groupadd -g 1000 nonroot && \
+  useradd -r -u 1000 -g nonroot nonroot
```

in our other images we are using `1001`, can we keep that consistent?
```suggestion
  useradd -r -u 1001 -g nonroot nonroot
```